content/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -66,268 +73,419 @@ The SDK is available in two formats: an NPM module and a CDN-hosted library.
+| `code_challange_method` | no       | `string`  | Define the code challenge method. Possible values: `S256` or `plain`; default: `S256`                                                                                                                                     |
+
+**Available class methods:**
+
+- [popup](#popup)
+- [redirect](#redirect)
+- [authorizeURL](#authorizeurl)
+- [verify](#verify)
+
+### popup  
+
+Returns a `Popup` object instance built on top of `sdk`.
+
+| Parameter             | Data type | Description                                                     |
+| --------------------- | --------- | --------------------------------------------------------------- |
+| [`options`](#options) | `object`  | An object with the same parameters as provided in `AccountsSDK` |
+
+```js
+  const popup = instance.popup();
 ```
 
-Insert the JavaScript code you can see above **before the closing `</body>` tag**.
+### redirect  
 
-That's it!
+Returns a `Redirect` object instance built on top of `sdk`.
 
-Your users will see the "Sign in with LiveChat" button if they are not yet logged in or they didn't grant access for your app.
+| Parameter             | Data type | Description                                                     |
+| --------------------- | --------- | --------------------------------------------------------------- |
+| [`options`](#options) | `object`  | An object with the same parameters as provided in `AccountsSDK` |
 
-If they are already logged in, you will immediately receive valuable information about your user, such as their `access_token` or `license` number.
+```js
+  const redirect = instance.redirect();
+```
 
-<Warning>
+### authorizeURL
 
-[**Don't forget to test the give consent step**](/monetization/app-review-process/#common-problems). Use `prompt: "consent"` to force the app to **ask you for access** to certain resources. It’s necessary for you to test the app as if you were a user who installs the app from Marketplace.
+Creates an authorization URL for the given flow and parameters.
 
-</Warning>
+| Parameter             | Data type | Description                                                     |
+| --------------------- | --------- | --------------------------------------------------------------- |
+| [`options`](#options) | `object`  | An object with the same parameters as provided in `AccountsSDK` |
+| `flow`                | `string`  | `code` or `token`                                               |
 
-# SDK documentation
+```js
+  const authURL = instance.authorizeURL();
+```
 
-## Methods
+### verify
 
-### **accountsSdk.init({ ... })**
+Verifies if the `state` parameter from the redirect matched the one provided upon initialization, If it does, the method returns `Transaction`. Otherwise, it returns `null`.
```

Comment:
```suggestion
Verifies if the `state` parameter from the redirect matched the one provided upon initialization, If it does, the method returns `Transaction`, which is used under the hood for `state` verification. Otherwise, it returns `null`.
```

Reply:
show a sample transaction?